crates/nu-explore/src/commands/try.rs

```diff
@@ -0,0 +1,70 @@
+use std::io::Result;
+
+use nu_protocol::{
+    engine::{EngineState, Stack},
+    Value,
+};
+
+use crate::{pager::TableConfig, views::InteractiveView};
+
+use super::{HelpExample, HelpManual, ViewCommand};
+
+#[derive(Debug, Default, Clone)]
+pub struct TryCmd {
+    command: String,
+    table_cfg: TableConfig,
+}
+
+impl TryCmd {
+    pub fn new(table_cfg: TableConfig) -> Self {
+        Self {
+            command: String::new(),
+            table_cfg,
+        }
+    }
+
+    pub const NAME: &'static str = "try";
```

Comment:
Unfortunately, we just added `try` in 0.72 as a part of `try { ... } catch { ... }`. Could you pick a new name?

Reply:
It's unrelated to it.

It's just a command name, so you can type `:try`.